packages/@ember/application/globals-resolver.js

```diff
@@ -93,9 +94,19 @@ class DefaultResolver extends EmberObject {
 
     @property namespace
     @public
+    @deprecated
   */
 
   init() {
+    deprecate(
+      'Using the globals resolver is deprecated. Use the ember-resolver package instead. See https://deprecations.emberjs.com/v3.x#toc_ember-deprecate-globals-resolver',
+      false,
+      {
+        until: '4.0.0',
+        id: 'globals-resolver',
+      }
+    );
```

Comment:
We need to add a `url` property here pointing to the deprecation guide URL for this.

Reply:
Fixed.